tests/test_meshing_workflow.py

```diff
@@ -576,3 +576,19 @@ def test_new_2d_meshing_workflow(new_mesh_session):
     # Switch to solution mode
     solver = meshing.switch_to_solver()
     assert solver
+
+
+@pytest.mark.fluent_version(">=23.2")
+def test_high_level_objects(new_mesh_session):
```

Comment:
This test is fine as far as it goes but it only tests the existence of a singleton object at initialisation time. As pointed out in my other comment, there is more to test here.

Reply:
@seanpearsonuk, sure. We can have a meeting once @mkundu1 is available today?